apps/asap-server/.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   rules: {
     'no-use-before-define': 'off',
     'no-unused-vars': 'off',
+    'lines-between-class-members': 'off',
```

Comment:
the reason i want to remove it is because it moans about class properties not being separated with spaces which is crazy imho

Reply:
Yeah, I don't understand this rule either